tests/e2e_runner.js

```diff
@@ -74,7 +77,7 @@ const testsToRun = allSetups
       return argv._.some(argName => {
         return path.join(process.cwd(), argName) == path.join(__dirname, 'e2e', name)
             || argName == name
-            || argName == name.replace(/\.ts$/, '');
+            || name.includes(argName);
```

Comment:
Recently we added 2 tests for (sorry for confusion) `ng test`. One called `test`, and the other called `test-assets`.

So, if I use `test-assets` as argument, would that match both tests according to condition?